### DISCUSSION TITLE
Wanting to upgrade laser module

### DISCUSSION THREAD

**Discussion by AzBikerD** [arkypita/LaserGRBL#2287] [Q&A]
Quite a while ago I purchased an off brand cheap laser etcher but it had a large working area. It has been fun and works good. It has a GRBL 1.1 board and a 700 mW laser module. I would like to upgrade to a 40W module with air assist to etch on metals and cut wood. Do  I need to upgrade my board to accommodate the air? If so what board would I need? What module would you recommend? Also it is a 12V module. This is the module I am currently looking at. [https://www.amazon.com/dp/B0BQ74TW14/?coliid=I99LLSX50PZGU&colid=3GCWJGJRNGRV2&ref_=list_c_wl_lv_ov_lig_dp_it&th=1](url)
Thanks for any help!

**Comment by arkypita** [comment]
First of all it is right to underline that the laser you linked to me is only 5W optical. I write this because there are also 40W optical lasers, but they are a completely different story.

![image](https://github.com/arkypita/LaserGRBL/assets/8782035/6d8e199e-1660-4764-8d42-217a51ff5a24)

In general you won't have any major problems doing this update, you just need to make sure that the power supply you use is capable of delivering at least 2A. If it doesn't, you need to replace it with a more powerful one.

If the electrical cables are very small, perhaps it is time to replace them with electrical cables capable of carrying more heat

Yes, you have to choose a 12V laser model because your control board is probably made for that voltage, newer lasers are 24V and would not work.

I don't know where you're writing from, but if you're in Europe, know that just in case I have several lasers for sale like new, used only for reviews, and I also have a 20W optical one for sale at an excellent price. This would really be the game changer for you.

https://www.ebay.it/itm/116217882320

**Reply by arkypita** [reply]
PS: forget to etch metals with 5W, it's not impossible but it is really hard. You need at least 10W (optical power speaking)

**Comment by AzBikerD** [comment]
Thank you so much for the useful information. I really do appreciate the time you take to help people out. I do have another question.    Looking around now for a 10W optical output module there are 40W and 80W versions. Is there an advantage or disadvantage to either?
  

  

  

  
  
  
  
  
>

**Reply by arkypita** [reply]
The only thing that matters is optical power.
A 5W optical power laser consumes approximately 20W of electrical power. Lasers with 10W of optical power usually have two paired 5W diodes, therefore they consume 40W of electrical power.

Anyone who claims 10W of optical power as "equivalent to 80W" is simply lying and doing so for marketing purposes.

Only one thing you need to look carefully at: many lasers have a rectangular focus point, while some have lenses that serve to make it square. Square is better, because it cuts with the same power in both directions.
